pom.xml

```diff
@@ -69,7 +69,12 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>publish-over</artifactId>
-            <version>0.18</version>
+            <version>0.19-SNAPSHOT</version>
```

Comment:
I just build locally and noticed this. Did you have a PR that needs to be done for the publish-over-plugin as well?

Reply:
Hi, no I do not have a PR for the parent publish-over-plugin. Everything I needed was already on master (mainly the version update of Jenkins dependencies). It just needs a release.
BR Tobias

Reply:
Ok, I'll do a release of that soon and then update this PR with the correct version and merge.